test/geometry/test_depth_warper.py

```diff
@@ -151,7 +151,7 @@ def test_compute_subpixel_step(self, batch_size, device, dtype):
 
         # test compute_subpixel_step
         subpixel_step = warper.compute_subpixel_step()
-        assert pytest.approx(subpixel_step.item(), 0.3536)
+        assert subpixel_step.item() == pytest.approx(0.3536)
```

Comment:
this can defenitely be replaced by assert_close
https://github.com/kornia/kornia/blob/master/test/color/test_gray.py#L7